integration/grpc/src/main/java/io/seata/integration/grpc/interceptor/server/ServerListenerProxy.java

```diff
@@ -15,25 +15,30 @@
  */
 package io.seata.integration.grpc.interceptor.server;
 
+import java.util.Map;
+import java.util.Objects;
+
 import io.grpc.ServerCall;
 import io.seata.common.util.StringUtils;
 import io.seata.core.context.RootContext;
-
-import java.util.Objects;
+import io.seata.core.model.BranchType;
 
 /**
  * @author eddyxu1213@126.com
  */
 public class ServerListenerProxy<ReqT> extends ServerCall.Listener<ReqT> {
 
     private ServerCall.Listener<ReqT> target;
-    private String xid;
+    private final String xid;
```

Comment:
Why is xid not integrated into the context

Reply:
xid is required, context is not required and may be empty.